### PR TITLE
gitAndTools.git-chglog: init at 0.9.1

### PR DESCRIPTION
**Pull request by ldenefle**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4778,6 +4778,12 @@
     githubId = 20250323;
     name = "Lucio Delelis";
   };
+  ldenefle = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4781]: Can you put this hunk into a seperate commit before the update named ``maintainers: add ldenefle``?

File: pkgs/applications/version-management/git-and-tools/git-chglog/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  goPackagePath = "github.com/git-chglog/git-chglog";
+
+  src = fetchFromGitHub {
+    owner = "git-chglog";
+    repo = "git-chglog";
+    rev = version;
+    sha256 = "08x7w1jlvxxvwnz6pvkjmfd3nqayd8n15r9jbqi2amrp31z0gq0p";
+  };
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-chglog/default.nix:16]:
```suggestion
  meta = with lib; {
```

And change the input to only lib.

File: pkgs/applications/version-management/git-and-tools/git-chglog/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    description = "CHANGELOG generator implemented in Go (Golang).";
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-chglog/default.nix:17]:
```suggestion
    description = "CHANGELOG generator implemented in Go (Golang)";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (ldenefle) [reply maintainers/maintainer-list.nix:4781]: sure

File: pkgs/applications/version-management/git-and-tools/git-chglog/default.nix (hunk 2 above)
Comment (ldenefle) [reply pkgs/applications/version-management/git-and-tools/git-chglog/default.nix]: done

File: pkgs/applications/version-management/git-and-tools/git-chglog/default.nix (hunk 3 above)
Comment (ldenefle) [reply pkgs/applications/version-management/git-and-tools/git-chglog/default.nix]: done

**Comment by roberth** [comment]
Thank you @ldenefle!

**Pull request merged**